tensorrt_llm/executor/worker.py

```diff
@@ -473,6 +488,11 @@ def _deduce_max_tokens(request: GenerationRequest,
                 context_phase_params=context_phase_params,
                 type=request_type)
 
+            if self._is_pytorch_backend:
+                # For PyTorch backend, attach the raw multimodal data
+                if request.multimodal_params is not None and request.multimodal_params.multimodal_data:
+                    executor_request.py_multimodal_data = request.multimodal_params.multimodal_data
+
```

Comment:
Can we merge `mm_embedding` and `mrope_config` into `mm_data` and skip their pybind interface for the PyT backend? Or, are they still required by the cpp runtime (e.g. batch manager)?

Reply:
Thanks @yechank-nvidia